common/user/userValidator.js

```diff
@@ -0,0 +1,29 @@
+const R = require('ramda')
+
+const {validate, validateRequired } = require('../validation/validator')
+
+const User = require('../../common/user/user')
+
+const errorKeys = {
+  invalidEmail: 'invalidEmail'
+}
+
+const getProp = (propName, defaultValue) => R.pathOr(defaultValue, propName.split('.'))
+
+const validateEmail = (propName, item) => {
+  const email = getProp(propName)(item)
+  return email && !User.validEmail(email) ? { key: errorKeys.invalidEmail } : null
```

Comment:
should we move the code User.validEmail here ?

Reply:
Ok, moved